project/build.properties

```
@@ -1 +1 @@
-sbt.version=1.5.7
+sbt.version=1.6.0
```

You can probably go straight to 1.6.2 I should think!

```suggestion
sbt.version=1.6.2
```